test-clients/src/main/java/com/hedera/services/bdd/spec/queries/meta/HapiGetTxnRecord.java

```diff
@@ -76,6 +85,12 @@
 	boolean assertNothingAboutHashes = false;
 	boolean lookupScheduledFromRegistryId = false;
 	boolean omitPaymentHeaderOnCostAnswer = false;
+	boolean validateAccountAmountsInTransferList = false;
+	boolean validateTokenAmountsInTokenTransferList = false;
+	boolean validateEntitiesInAssessedCustomFees = false;
+	List<Pair<String, Long>> accountAmountsToValidate = new ArrayList<>();
+	List<Triple<String, String, Long>> tokenAmountsToValidate = new ArrayList<>();
+	List<Triple<String, String, Long>> assessedCustomFeesToValidate = new ArrayList<>();
```

Comment:
nit: if `accountAmountsToValidate`, `tokenAmountsToValidate`, and `assessedCustomFeesToValidate` can be defined as `Optional` and initialized as `Optional.empty()`, the `validateAccountAmountsInTransferList...` and other two booleans won't be needed.

Reply:
No, @ljianghedera, @anighanta, we should not use `Optional`. The booleans can be removed by checking whether each list is empty or not.
The correct fix is here https://github.com/hashgraph/hedera-services/pull/1840/commits/49a9fa01b47abcbf16702dca68e261541d03a865.